explorer/testdata/function/fail_return_type_is_call.carbon

```diff
@@ -0,0 +1,20 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// RUN: %{not} %{explorer} %s 2>&1 2>&1 | %{FileCheck} %s
+// AUTOUPDATE: %{explorer} %s
+
+package EmptyIdentifier impl;
+
+fn apply[T:! Type, U:! Type](f: T, EmptyIdentifier: U)
+{
+// CHECK:COMPILATION ERROR: {{.*}}/explorer/testdata/function/fail_return_type_is_call.carbon:[[@LINE+1]]: expected a tuple
```

Comment:
Maybe this would more accurately be `fail_return_call_has_invalid_body`?

Reply:
Done.